common/interfaces/user_interface/ui_confirmation.c

```diff
@@ -190,7 +198,12 @@ static void cancel_btn_event_handler(lv_obj_t *cancel_btn,
     case LV_EVENT_DEFOCUSED:
       lv_btn_set_state(cancel_btn, LV_BTN_STATE_REL);
       break;
-
+    case LV_EVENT_DELETE: {
+      /* Destruct object and data variables in case the object is being deleted
+       * directly using lv_obj_clean() */
+      confirm_scr_destructor();
```

Comment:
in `cancel_btn_event_handler` replace `confirm_scr_destructor` with `lv_obj_clean` (at line 196).

Reply:
Fixed